pkg/spoke/submarineragent/config_controller.go

```diff
@@ -51,6 +51,7 @@ const (
 	gcpZoneLabel           = "failure-domain.beta.kubernetes.io/zone"
 )
 
+// TODO: Are these hard-coded credntials okay or an issue?
```

Comment:
Taking a second look at this failure, I'm less confident in assuming it's okay. What do you think @skitt?

Reply:
They’re not hard-coded credentials, they’re a reference to a K8s secret which contains credentials.